### PR TITLE
Cli query commands for UI signing

### PR DESCRIPTION
**Pull request by aristotleee**

### CODE REVIEW

File: src/bridge/transactions/peg_in_confirm.rs [hunk 1]
```diff
@@ -128,6 +141,15 @@ impl PegInConfirmTransaction {
         push_taproot_leaf_unlock_data_to_witness(&mut self.tx, input_index, unlock_data);
```
Comment (StarDumpling) [inline src/bridge/transactions/peg_in_confirm.rs]: We could reuse `push_depositor_signature_input_0()` function here.

Just call `self.push_depositor_signature_input_0(schnorr_signature);`

File: src/bridge/transactions/signing.rs [hunk 2]
```diff
@@ -300,3 +314,20 @@ pub fn populate_taproot_input_witness_default(
...
+    signatures: &Vec<bitcoin::taproot::Signature>,
+) {
+    let unlock_data = signatures.iter().map(|sig| sig.to_vec()).collect();
+    push_taproot_leaf_unlock_data_to_witness(tx, input_index, unlock_data);
+    push_taproot_leaf_script_and_control_block_to_witness(
+        tx,
+        input_index,
+        taproot_spend_info,
+        script,
+    );
```
Comment (StarDumpling) [inline src/bridge/transactions/signing.rs]: Just call `populate_taproot_input_witness` function instead
```suggestion
    populate_taproot_input_witness(
        tx,
        input_index,
        taproot_spend_info,
        script,
        unlock_data,
    );
```

File: src/bridge/transactions/peg_in_confirm.rs (hunk 1 above)
Comment (aristotleee) [reply src/bridge/transactions/peg_in_confirm.rs]: Fixed as suggested

File: src/bridge/transactions/signing.rs (hunk 2 above)
Comment (aristotleee) [reply src/bridge/transactions/signing.rs]: Fixed as suggested

**Pull request merged**